### PR TITLE
Swisson manufacturer update

### PR DESCRIPTION
**Pull request by fischbas**
Added a manufacturer specific PID and updated product information.

[Swisson_PID_and_products.pdf](https://github.com/OpenLightingProject/rdm-app/files/4629528/Swisson_PID_and_products.pdf)

### CODE REVIEW

File: data/node_data.py [hunk 1]
```diff
@@ -74,4 +74,10 @@
      'image_url': 'http://www.luminex.be/products/0100037/photo_037.png',
      'tags': ['Art-Net']},
   ],
+  0x5377: [
+    {'name': 'XND Ethernet DMX Node',
+     'link': 'https://swisson.com/en/products/data-distribution/ethernet-dmx-node/',
+     'image_url': 'https://swisson.com/site/assets/files/1056/xnd-4_box_and_rack_and_xnd-8.500x0.png',
+     'tags': ['tool']},
```
Comment (peternewman) [inline data/node_data.py:81]: I think Art-Net would be more appropriate here (and possibly sACN too).

Comment (fischbas) [reply data/node_data.py]: Thanks for catching that copy/paste error!

**Comment by fischbas** [comment]
Feel free to merge if it still looks good to you!
Thanks a lot!

File: data/pid_data.py [hunk 2]
```diff
@@ -2699,6 +2699,22 @@
...
+   'pids': [
+            # SWPID_AUTO_UNIVERSE_NUMBER
+            {'name': 'SWPID_AUTO_UNIVERSE_NUMBER',
+             'get_request': {'items': []},
+             'get_response': {'items': [{'name': 'universe_number', 'type': 'uint16'}]},
+             'get_sub_device_range': 0,
+             'set_request': {'items': [{'name': 'universe_number', 'type': 'uint16'}]},
+             'set_response': {'items': []},
+             'set_sub_device_range': 0,
+             'value': 0x8082,
```
Comment (peternewman) [inline data/pid_data.py:2714]: Does this mean you've got a load of other PIDs too, or are they private?

Comment (fischbas) [reply data/pid_data.py:2714]: No , we haven't got any other private PIDs, just tried to comply with the specification:

> Manufacturer-Specific PID values should be selected by choosing the appropriate category fromTable A-3 and adding an offset of 0x8000 to preserve a logical organization. 

E1-20 6.2.10.2

In our case we used PID DEVICE_LABEL + 0x8000.

Comment (peternewman) [reply data/pid_data.py:2714]: Well I'm glad someone follows that bit of the spec, I think you might be one of the only ones, even we don't do that! :laughing:

**Pull request merged**